tests/ossfuzz/fuzz_encode_raw.c

```diff
@@ -0,0 +1,53 @@
+///////////////////////////////////////////////////////////////////////////////
```

Comment:
We can also omit this fuzz target. It should be covered by `fuzz_encode_alone`, which is mostly covered by `lzma_encode_stream()` anyway